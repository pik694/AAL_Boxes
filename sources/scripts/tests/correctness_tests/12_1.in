12
67 24 32
79 77 54
59 38 28
98 66 36
18 57 55
5 71 52
100 96 19
35 66 86
2 77 38
19 41 69
97 87 69
91 100 53

