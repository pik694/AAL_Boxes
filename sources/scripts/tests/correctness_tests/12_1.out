8 5 0 7 9 3 11 4 6 2 1 10 
2255706->1469963
